### PR TITLE
added scripts to build docker images and publish it using Travis

### PR DESCRIPTION
**Pull request by shsethi**

### CODE REVIEW

File: scripts/publish-to-docker-hub.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (djsutho) [reply scripts/publish-to-docker-hub.sh]: Is this the same file as here https://github.com/ExpediaDotCom/haystack-trends/blob/master/scripts/publish-to-docker-hub.sh At some point we should try to find a places to make these common.

File: scripts/publish-to-docker-hub.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+set -e
+
+QUALIFIED_DOCKER_IMAGE_NAME=$DOCKER_ORG/$DOCKER_IMAGE_NAME
+echo "DOCKER_ORG=$DOCKER_ORG, DOCKER_IMAGE_NAME=$DOCKER_IMAGE_NAME, QUALIFIED_DOCKER_IMAGE_NAME=$QUALIFIED_DOCKER_IMAGE_NAME"
+echo "BRANCH=$BRANCH, TAG=$TAG, SHA=$SHA"
+
+# login
+docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
```
[the remark this answers is not in the document]
Comment (shsethi) [reply scripts/publish-to-docker-hub.sh]: Yes it is set in travis UI

File: scripts/publish-to-docker-hub.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (shsethi) [reply scripts/publish-to-docker-hub.sh]: yes its the same file

File: kafka/docker/scripts/common.sh [hunk 3]
```diff
@@ -33,9 +35,15 @@ while :; do
...
             package=1
             ;;
+        -b|--build)
+            build=1
+            ;;
         -r|--run)
             run=1
             ;;
+        --release)
+            release=1
```
[the remark this answers is not in the document]
Comment (djsutho) [reply kafka/docker/scripts/common.sh]: Fixed

**Pull request merged**